CMakeLists.txt

```diff
@@ -277,6 +277,7 @@ list(APPEND EXPORT_MODULES
     "cmake/AwsSIMD.cmake"
     "cmake/AwsFindPackage.cmake"
     "cmake/AwsFeatureTests.cmake"
+    "cmake/AwsCrtLoadTarget.cmake"
```

Comment:
I'm worried that an application with a dead-simple CMakeLists.txt file won't find this file

test it out